Create adopters.md (#835)
flatcar/flatcar: adopters.md

      
Public anouncmentce: 

    * N: AT&T (DirectTV) 
      L: https://medium.com/cloud-native-the-gathering/certified-kubernetes-administrator-join-our-team-its-a-good-thing-7e27ab34dc88
      D: "We are integrating Flatcar Container Linux, Istio, OPA, Multi-Region, KNative, and so many other technologies and concepts it makes the mind hurt a bit."

    * N: Digital Science 
      L: https://digital-science.com
      D: We're running Flatcar on all our self-hosted Kubernetes clusters on AWS, used for all data processing behind Dimensions (https://dimensions.ai). We choose Flatcar for security and simplicity.
      C: soren [at] uberresearch.com

    * N: Giant Swarm 
       L: Https://www.giantswarm.io/blog/time-to-catch-a-new-train-flatcar-linux| 

    * N: Jérôme Petazzoni
      L: https://twitter.com/jpetazzo/status/1317903569081741312 
      C: https://twitter.com/jpetazzo/

    * N: Mettle
      L https://swade1987.medium.com/upgrading-to-flatcar-linux-746751e89ab4

    * N: The @ Company 
      L: https://twitter.com/cpswan/status/1534481517887512577?s=20&t=ODnO_TPa4nhC62KNAB9Stw
      C: Chris Swan https://twitter.com/cpswan/

> 

Reviewer: ```suggestion
    * N: STACKIT
      D: Flatcar is used in our Kubernetes as a Service (KaaS) offering called SKE 
      L: https://www.stackit.de/de/produkt/stackit-kubernetes-engine/
      C: [info@stackit.de](mailto:info@stackit.de)
```

Reviewer: Hi @miao0miao, added us (STACKIT / Schwarz IT) as suggestion

Who is using Flatcar?
====================

> Sharing experiences and learning from other users is essential. While the [Flatcar Slack community](https://kubernetes.slack.com/archives/C03GQ8B5XNJ) allows

Reviewer 2: ```suggestion
The following is a list of adopters who have publicly spoken about their use of Flatcar, or who have added themselves to this list.
```

> users to get in touch and also [the Matrix channel](https://app.element.io/#/room/#flatcar:matrix.org) The following is a directory of adopters to help identify users of individual features. The users themselves directly maintain the list.

Reviewer 2: ```suggestion
```

Adding yourself as a user
-------------------------

> If you are using Flatcar, please consider adding yourself as a user with a quick description of your use case by opening a pull request to this file and adding a section describing your usage of Flatcar. If you are open to others contacting you about your use of Flatcar on Slack or Matrix, add your Slack/Matrix nickname as well.

Reviewer 2: ```suggestion
If you are using Flatcar, please consider adding yourself as a user with a quick description of your use case by opening a pull request to this file and adding a section describing your usage of Flatcar. If you are open to others contacting you about your use of Flatcar on Slack or Matrix, include your Slack/Matrix nickname or email as well.
```

    
    * N: Name of user (company or individual) 
      D: Description 
      L: Link with further information (optional) 
      C: Contacts available for questions (optional)

Example entry:

    * N: Flatcar Example User Inc.

>       D: Flatcar Example User Inc. is using Flatcar 

Reviewer 2: ```suggestion
      D: Using Flatcar for running Kubernetes in Azure via Cluster API
```

>       L: https://www.flatcar.org/ 

Reviewer 2: ```suggestion
      L: https://www.exampleuser.com 
```

>       C: @slacknick and/or @Matrixnick 

Reviewer 2: ```suggestion
      C: Slack: @slacknick and/or Matrix: @Matrixnick and/or Email: nick [at] exampleuser [dot] com
```

    

Requirements to be listed
-------------------------

 * You must represent the user listed. Do not add entries on behalf of
   other users.

> * Commercial or production use is not required a user can be an end user, cloud service provider or consultant as long as it is permanent deployment and not a trail deployment. A well-done home lab setup can be equally

Reviewer 2: ```suggestion
* Commercial or production use is not required. A user can be an end user, cloud service provider or consultant as long as it is a permanent deployment and not a trial deployment. A well-done home lab setup can be equally
```

   interesting as a large-scale commercial deployment.

Users 
-----
 

>     * N: Adobe

Reviewer 2: this seems very sparse. can't we use our contacts to get a quote?

Reviewer 3: Does this work for you?

```
D: Adobe runs Flatcar on over 18,000 nodes in our fleet of Kubernetes clusters across multiple cloud providers and private data centers in 22 different regions worldwide.
```

For contacts you can put Mike Tougeron (Slack: `@Mike Tougeron`) and Tony Gosselin (Slack: `@Tony Gosselin`).

    
    * N: AloPeyk
      D: We are an on-demand delivery business that every day serve millions of requests on a consistent bare metal infrastructure have this concern to choose a reliable and atomic operations system but none of the common OS couldn't satisfy us for such heavy workloads. Since we migrated our production Kubernetes cluster nothing can break this consistent and solid cluster which is powered by amazing Flatcoar OS.
    
    * N: Cloud house
      D: Flatcar OS has been useful for us for our on-premise solution to our customers 
      

>     * N: Equinix Metal

Reviewer 2: can't we get our EM contacts to suggest a contact to list?

Author: @ahrkrak  I did not get an answer yet

> 

Reviewer 2: ```suggestion
L: https://kinvolk.io/blog/2021/02/case-study-equinix-metal-builds-on-flatcar/
D: Equinix uses Flatcar as the OS for its bare metal cloud control plane, which runs in Kubernetes
```

      
    * N: Finleap Connect
      D: Finleap Connect - At finleap connect we serve over a million financial transactions per day. As a regulated company using a pure cloud-native stack based on Kubernetes, using Flatcar as our foundational building block for reliable, secure and immutable nodes across the public clouds and on bare-metal deployments was a day one decision we never regretted. Today Flatcar serves all of our 12 production clusters with over 300 nodes on public clouds in 3 countries and on our bare-metal private-cloud setup.
    

>     * N: Host with quantum 

Reviewer 4: ```suggestion
    * N: Planetary Quantum GmbH
```

>       C: https://twitter.com/klimpong

Reviewer 4: Can I PR some changes to this? 😆

Reviewer 2: Yes please do!!

Reviewer 5: @till please feel free to [suggest](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) right here in the PR if that works for you!

>       C: https://twitter.com/klimpong

Reviewer 4: ```suggestion
      L: https://www.planetary-quantum.com/
      D: Planetary Quantum is a berlin-based provider of Docker-hosting and application hosting. Our sister company Planetary Networks colocates their private cloud in two (fiber-)interconnected datacenters in Berlin and Quantum offers container-based solutions (Docker Swarm and a custom tailored application hosting) on top of Flatcar Linux. Flatcar Linux is a great choice for us because it's a modern Linux, well-suited for Docker and Kubernetes due to recent versions of Kernel, SystemD, immutable root and a well-tested userland. Simple and straight-forward updates of the OS make running Flatcar a no-brainer for us. We currently operate over 50 clusters for our customers in our private cloud — all based on Flatcar Linux.
```

>     * N: Intersys AG 

Reviewer 2: can we get a description?

>     * N: Spinoco

Reviewer 2: can't we get at least a description from Spinoco?

Author: done

    * N: Wipro
      D: Wipro Business Solutions uses Flatcar Linux to power their hybrid/multi-cloud PostgreSQL containerized DBaaS platform. Each provisioned database is running on a dedicated lightweight stack with Flatcar Linux as the foundational OS running on each database VM. In addition the DBaaS API itself and all supporting machines use Flatcar Linux as well. Flatcar Linux has proven to be a well-supported rocket solid OS with minimal attack surface, build in update mechanism and integrated docker daemon. Ignition brings in an early boot provisioning utility that perfectly adds to the full automation approach of the PostgreSQL DBaaS platform. We use it at scale on-prem with OpenStack cloud but also with public clouds like Google and Tencent
      
Public anouncmentce: 

    * N: AT&T (DirectTV) 
      L: https://medium.com/cloud-native-the-gathering/certified-kubernetes-administrator-join-our-team-its-a-good-thing-7e27ab34dc88

> 

Reviewer 2: ```suggestion
D: "We are integrating Flatcar Container Linux, Istio, OPA, Multi-Region, KNative, and so many other technologies and concepts it makes the mind hurt a bit."
```

>     * N: Darren Shepherd 

Reviewer 2: please delete. this is an endorsement, not usage.

      L:|https://twitter.com/ibuildthecloud/status/1288498089574055938 
      C: https://twitter.com/ibuildthecloud 

>     * N: Digital Sceince 

Reviewer 2: ```suggestion
    * N: Digital Science 
```

>       L: https://www.linkedin.com/in/s%C3%B8ren-dalby-larsen-5502141/ 

Reviewer 2: this is just a link to the guy's LinkedIn. Can you find more? Why is he listed here?

>     * N: Digital Science 

Reviewer 6: N: Digital Science
L: https://digital-science.com
D: We're running Flatcar on all our self-hosted Kubernetes clusters on AWS, used for all data processing behind Dimensions (https://dimensions.ai). We choose Flatcar for security and simplicity.
C: soren [at] uberresearch.com

>       L: https://www.linkedin.com/in/s%C3%B8ren-dalby-larsen-5502141/ 

Reviewer 6: I don't need this, so please delete.

      C: Søren Dalby Larsen 

    * N: Giant Swarm 
       L: Https://www.giantswarm.io/blog/time-to-catch-a-new-train-flatcar-linux| 

>     * N: Jérôme Petazzoni

Reviewer 2: Please check with Jerome. I don't think the tweet counts as him saying he's using it. He provides training, so that is probably not true.

Author: @ahrkrak I did not get an answer yet

Reviewer 2: @miao0miao ok, please just remove

      L: https://twitter.com/jpetazzo/status/1317903569081741312 
      C: ttps://twitter.com/jpetazzo/

>     * N: Kelsey Hightower

Reviewer 2: this is an endorsement, not a user. please delete

      L: https://twitter.com/RiRa12621/status/1283119809484513282 
      C: https://twitter.com/kelseyhightower
    
    * N: Mettle
      L https://swade1987.medium.com/upgrading-to-flatcar-linux-746751e89ab4

>     * C: Chris Swan

Reviewer 2: delete, the is covered under the @ Company below
